README.md

```diff
@@ -55,6 +55,12 @@ Please refer to the following instructions to setup Oppiabot for the first time
   ```
 
 4. The Oppiabot uses environment variables. These are configured in the server settings. To deploy the bot locally, create a `.env` file and copy the contents of `.env.example` to it. You will need to adjust these variables accordingly following the instructions in the subsequent steps. Run following command to copy `.env.example` to `.env`
+
+If using windows cmd and Say you have made folder as instructed in point 1 and on Desktop then run below command, Replace <username> with Name of your computer.
+``` bash
+copy C:\Users\<username>\Desktop\opensource\oppiabot\.env.example  C:\Users\<username>\Desktop\opensource\oppiabot\.env 
```

Comment:
This makes a lot of assumptions about where the folder is stored.
We can just tell the user to copy `.env.example` to a new file named `.env`

Reply:
Sir Are you suggesting that line after "Say" should be removed?

Reply:
Actually Windows use Absolute Path in its cmd, So I thought it will be best if we clarify the user. Please let me know if you want it removed. I will remove it right then.

Reply:
Sir, Changes have been done and now no assumption is being made. Thanks.